docs/update-cli.md

```diff
@@ -0,0 +1,26 @@
+# Update OpenAPI CLI
+
+If you've already installed OpenAPI CLI, you'll want to keep it as up to date as possible. Staying on top of versions ensures that you get the latest functionality and bug fixes. Depending on how you originally installed OpenAPI CLI, there are two ways to update it.
+
+:::success Tip
+Before you begin, ensure that you have the latest version of npm installed.
+:::
+
+## How to update your openapi-starter installation
+
+If you installed OpenAPI CLI using the [openapi-starter template](https://github.com/Redocly/openapi-starter) follow these steps:
+
+1. Navigate to and open `package.json` in any text editor.
+2. Find the line that contains `"@redocly/openapi-cli":` followed by the version number.
+3. Change the version number to the latest one in the [Changelog](https://redocly.com/docs/cli/changelog/).
+4. Save the changes then close `package.json`.
+
```

Comment:
Confirming project-level installation is different form global installation. It will need an example of how to do that here.

```suggestion
Check the project-level installation by running this command from the project's root directory: `npx redocly --version`.
```